man/systemd.exec.xml

```diff
@@ -545,6 +545,13 @@
         To disable the safety check that the extension-release file name matches the image file name, the
         <varname>x-systemd.relax-extension-release-check</varname> mount option may be appended.</para>
 
+        <para>This option can be used together with a <option>notify-reload</option> service type and
+        <citerefentry><refentrytitle>systemd.v</refentrytitle><manvolnum>7</manvolnum></citerefentry>
+        to manage configuration updates. When such a service carrying confext images is reloaded, the confext
+        itself will also be reloaded to pick up any changes. This only applies to confext extensions. See
+        <citerefentry><refentrytitle>systemd.service</refentrytitle><manvolnum>5</manvolnum></citerefentry>
+        also for details.</para>
```

Comment:
this is really confusing, as it somehow puts notify-reload into the mix, suggesting there was any reason for that, but that's an entirely made-up concept. let#s not complicate stuff by adding arbitrary limitations.

Reply:
Just to be sure, do we want to remove this limitation and make this behaviour the default for all service types altogether? Does it need a config knob to opt in or out?

Reply:
We do not want to remove this specialization, and we do not want yet-another-option. As already mentioned many times, it is completely unclear how this would even possibly work with arbitrary random processes/scripts/programs running in ExecReload and the semantics are completely unclear, given there is not even a real use case, but it's purely speculative. And it gets even more impossibly complicated once asyncronicity enter the picture. Hence let's not derail the actual, real use case that actually exists for something that nobody wants or needs. If and when an actual, real world use case appears, then we can reason about the exact semantics and how it could be integrated.